consumer.go

```diff
@@ -815,8 +815,14 @@ func (r *Consumer) onConnClose(c *Conn) {
 }
 
 func (r *Consumer) backoffDurationForCount(count int32) time.Duration {
-	backoffDuration := r.config.BackoffMultiplier *
-		time.Duration(math.Pow(2, float64(count)))
+	// Jittered Backoff
+	// http://www.awsarchitectureblog.com/2015/03/backoff.html
+	if 0 == r.backoffDuration {
+		return r.config.MinBackoffDuration
+	}
+	base := int(r.config.MinBackoffDuration)
+	sleep := int(r.backoffDuration)
+	backoffDuration := time.Duration((r.rng.Intn(sleep * 3) - base) + base)
```

Comment:
this implements the "decorrelated" algorithm from the article, since we already have this setup to be based on "attempt" (`count`) it might be best to use the "full jitter" algorithm, which uses attempts.

They seem to perform basically the same.

Alternatively, we should probably rename this function and remove its parameter since it isn't being used.